doc/introduction/base-oscillations.md

```diff
@@ -175,6 +175,14 @@ These parameters can be set for all base oscillations.
 | complexity  | Float | The number of bits used to generate the sequence. This controls the length of the repeating sequence and its complexity. |
 | smoothing   | Float | Smoothing factor for convolutional smoothing of the generated bit sequence (highly recommended). Default is 0.01.        |
 
+## Custom Input
+
```

Comment:
Can you add a small description here? Please add the constraints on the datasets as well:

- CSV with xxx headers or no header?
- no anomalies!
- ...